pages/search.js

```diff
@@ -262,6 +263,30 @@ class SearchPage extends React.Component {
     });
   };
 
+  i18nCountryName = (intl, country) => {
+    const countryNames = new Intl.DisplayNames(intl.locale, { type: 'region' });
```

Comment:
Cool if we can do that! We probably don't need `i18n-iso-countries` in the codebase anymore then?